codespell_lib/data/dictionary.txt

```diff
@@ -6075,38 +6138,51 @@ initally->initially
 initation->initiation
 initators->initiators
 initiaitive->initiative
+initiales->initialize, initializes, initials, initialise, initialises,
+initialialize->initialize
+initialiase->initialise, initialize,
+initialiation->initialization
+initialied->initialized
+initialilzing->initializing
+initialisiert->initialised, initialized,
 initialiting->initializing
+initialitze->initialize
+initialitzing->initializing
+initializaing->initializing
+initializeing->initializing
 initializiation->initialization
-initialse->initialize
-initialsed->initialized
-initialses->initializes
+initializong->initializing
+initialse->initialize, initialize,
```

Comment:
did you mean `initialise` for the second one?

https://travis-ci.org/lucasdemarchi/codespell/jobs/376840463

Reply:
<facepalm>